main.go

```diff
@@ -69,11 +70,10 @@ func main() {
 	}
 
 	mgr, err := ctrl.NewManager(config, ctrl.Options{
-		Scheme:             scheme,
-		MetricsBindAddress: metricsAddr,
-		Port:               9443,
```

Comment:
Port parameter is missing, is it a API change?

Reply:
API change, the same with MetricsBindAddress, was substituted by MetricsServer